### PR TITLE
Video player breaking tests

### PR DESCRIPTION
**Pull request by JackHowa**
Getting dependency issue with video player. I guess it should be included in top-level package.json. 

Steps: 

- Clear block node modules `npx lerna clean`
- Clear top-level node_modules `rm -rf node_modules`
- Install top-level node_modules `npm i`
- Run tests `npm t`

```

  ● article-body chain › Render raw html correctly › should render raw html

    Cannot find module '@wpmedia/video-player-block' from 'default.jsx'

    However, Jest was able to find:
        './default.jsx'
        './default.test.jsx'

    You might want to include a file extension in your import, or update your 'moduleFileExtensions', which is currently ['js', 'json', 'jsx', 'ts', 'tsx', 'node'].

    See https://jestjs.io/docs/en/configuration#modulefileextensions-array-string

       6 | import getProperties from 'fusion:properties';
       7 | import getTranslatedPhrases from 'fusion:intl';
    >  8 | import VideoPlayer from '@wpmedia/video-player-block';
         | ^
       9 | import {
      10 |   Gallery, ImageMetadata, Image,
      11 | } from '@wpmedia/engine-theme-sdk';

      at Resolver.resolveModule (../../node_modules/jest-resolve/build/index.js:259:17)
      at Object.<anonymous> (chains/article-body/default.jsx:8:1)

  ● article-body chain › Render raw html correctly › should not render raw html

    Cannot find module '@wpmedia/video-player-block' from 'default.jsx'

    However, Jest was able to find:
        './default.jsx'
        './default.test.jsx'

    You might want to include a file extension in your import, or update your 'moduleFileExtensions', which is currently ['js', 'json', 'jsx', 'ts', 'tsx', 'node'].

    See https://jestjs.io/docs/en/configuration#modulefileextensions-array-string

       6 | import getProperties from 'fusion:properties';
       7 | import getTranslatedPhrases from 'fusion:intl';
    >  8 | import VideoPlayer from '@wpmedia/video-player-block';
         | ^
       9 | import {
      10 |   Gallery, ImageMetadata, Image,
      11 | } from '@wpmedia/engine-theme-sdk';

      at Resolver.resolveModule (../../node_modules/jest-resolve/build/index.js:259:17)
      at Object.<anonymous> (chains/article-body/default.jsx:8:1)

  ● article-body chain › Render interstital link correctly › should render interstitial link

    Cannot find module '@wpmedia/video-player-block' from 'default.jsx'

    However, Jest was able to find:
        './default.jsx'
        './default.test.jsx'

    You might want to include a file extension in your import, or update your 'moduleFileExtensions', which is currently ['js', 'json', 'jsx', 'ts', 'tsx', 'node'].

    See https://jestjs.io/docs/en/configuration#modulefileextensions-array-string

       6 | import getProperties from 'fusion:properties';
       7 | import getTranslatedPhrases from 'fusion:intl';
    >  8 | import VideoPlayer from '@wpmedia/video-player-block';
         | ^
       9 | import {
      10 |   Gallery, ImageMetadata, Image,
      11 | } from '@wpmedia/engine-theme-sdk';

      at Resolver.resolveModule (../../node_modules/jest-resolve/build/index.js:259:17)
      at Object.<anonymous> (chains/article-body/default.jsx:8:1)

  ● article-body chain › Render interstital link correctly › should not render interstitial link

    Cannot find module '@wpmedia/video-player-block' from 'default.jsx'

    However, Jest was able to find:
        './default.jsx'
        './default.test.jsx'

    You might want to include a file extension in your import, or update your 'moduleFileExtensions', which is currently ['js', 'json', 'jsx', 'ts', 'tsx', 'node'].

    See https://jestjs.io/docs/en/configuration#modulefileextensions-array-string

       6 | import getProperties from 'fusion:properties';
       7 | import getTranslatedPhrases from 'fusion:intl';
    >  8 | import VideoPlayer from '@wpmedia/video-player-block';
         | ^
       9 | import {
      10 |   Gallery, ImageMetadata, Image,
      11 | } from '@wpmedia/engine-theme-sdk';

      at Resolver.resolveModule (../../node_modules/jest-resolve/build/index.js:259:17)
      at Object.<anonymous> (chains/article-body/default.jsx:8:1)

  ● article-body chain › Render image correctly › should render image with figcaption and author

    Cannot find module '@wpmedia/video-player-block' from 'default.jsx'

    However, Jest was able to find:
        './default.jsx'
        './default.test.jsx'

    You might want to include a file extension in your import, or update your 'moduleFileExtensions', which is currently ['js', 'json', 'jsx', 'ts', 'tsx', 'node'].

    See https://jestjs.io/docs/en/configuration#modulefileextensions-array-string

       6 | import getProperties from 'fusion:properties';
       7 | import getTranslatedPhrases from 'fusion:intl';
    >  8 | import VideoPlayer from '@wpmedia/video-player-block';
         | ^
       9 | import {
      10 |   Gallery, ImageMetadata, Image,
      11 | } from '@wpmedia/engine-theme-sdk';

      at Resolver.resolveModule (../../node_modules/jest-resolve/build/index.js:259:17)
      at Object.<anonymous> (chains/article-body/default.jsx:8:1)

  ● article-body chain › Render image correctly › should not render image with figcaption and author

    Cannot find module '@wpmedia/video-player-block' from 'default.jsx'

    However, Jest was able to find:
        './default.jsx'
        './default.test.jsx'

    You might want to include a file extension in your import, or update your 'moduleFileExtensions', which is currently ['js', 'json', 'jsx', 'ts', 'tsx', 'node'].

    See https://jestjs.io/docs/en/configuration#modulefileextensions-array-string

       6 | import getProperties from 'fusion:properties';
       7 | import getTranslatedPhrases from 'fusion:intl';
    >  8 | import VideoPlayer from '@wpmedia/video-player-block';
         | ^
       9 | import {
      10 |   Gallery, ImageMetadata, Image,
      11 | } from '@wpmedia/engine-theme-sdk';

      at Resolver.resolveModule (../../node_modules/jest-resolve/build/index.js:259:17)
      at Object.<anonymous> (chains/article-body/default.jsx:8:1)


Test Suites: 1 failed, 1 skipped, 82 passed, 83 of 84 total
Tests:       23 failed, 3 skipped, 662 passed, 688 total
Snapshots:   2 passed, 2 total
Time:        22.871s, estimated 26s
Ran all test suites in 54 projects.
npm ERR! Test failed.  See above for more details.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -111,4 +112,4 @@
   "browserslist": [
     "defaults"
   ]
-}
+}
```
Comment (beltrancaliz) [inline package.json:115]: :(

Comment (JackHowa) [reply package.json:115]: hehe I don't know why but always happens on `npm i`

Comment (JackHowa) [reply package.json:115]: think I've looked it up before but forget

Comment (JackHowa) [reply package.json:115]: think I've looked it up before but forget

**Pull request merged**